components/style/values/computed/basic_shape.rs

```diff
@@ -196,21 +197,7 @@ impl ToCss for ShapeRadius {
     }
 }
 
-/// https://drafts.csswg.org/css-backgrounds-3/#border-radius
-#[derive(Clone, PartialEq, Copy, Debug)]
-#[cfg_attr(feature = "servo", derive(HeapSizeOf))]
-#[allow(missing_docs)]
-pub struct BorderRadius {
-    pub top_left: BorderRadiusSize,
-    pub top_right: BorderRadiusSize,
-    pub bottom_right: BorderRadiusSize,
-    pub bottom_left: BorderRadiusSize,
-}
+/// The computed value of `BorderRadius`
+pub type BorderRadius = GenericBorderRadius<LengthOrPercentage>;
 
-impl ToCss for BorderRadius {
-    #[inline]
-    fn to_css<W>(&self, dest: &mut W) -> fmt::Result where W: fmt::Write {
-        basic_shape::serialize_radius_values(dest, &self.top_left.0, &self.top_right.0,
-                                             &self.bottom_right.0, &self.bottom_left.0)
-    }
-}
+impl Copy for BorderRadius {}
```

Comment:
(as an aside, since it was pre-existing, I think we should avoid deriving copy on such big types. We've had memmove traffic before, and if we can avoid accidental memcpys it'd be lovely)